Stripe/NSURLComponents+Stripe.h

```diff
@@ -0,0 +1,37 @@
+//
+//  NSURLComponents+Stripe.h
+//  Stripe
+//
+//  Created by Brian Dorfman on 1/26/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSURLComponents (Stripe)
+
+
+/**
+ Returns or sets self.queryItems as a dictionary where all the keys are the item
+ names and the values are the values. When reading, if there are duplicate 
```

Comment:
Nope, reading is correct. When writing you can only have one of each key, but when reading it is converting from an array which allows dupes and has significant ordering to a dictionary which does not, so.